docs/concepts/whatisvpp/whatisvpp.rst

```diff
@@ -0,0 +1,363 @@
+.. _whatisvpp01:
+
+.. toctree::
+
+Introduction
+------------
+
+The VPP platform is an extensible framework that provides out-of-the-box
+production quality switch/router functionality. It is the open source
```

Comment:
1. We need remove _all_ references to Cisco, we should only talk about FD.io VPP.
2. We need to remove reference to FD.io VPP as a switch/router, we should only talk about it as a flexible network stack, that can be used as a switch, router, load balancer etc.